src/fhi/typography/_typography.scss

```diff
@@ -179,7 +179,7 @@ a {
     h1, .h1 {
         font-size: $fhi-core-h1-font-size-lg;
     }
-    body, p, .p, a {
+    body, p, .p {
```

Comment:
It forces overrides on anchor tags basically everywhere. I reset it now, and since this is a global issue it might be better to fix this as it's own issue. Typography in general might be ready for an overhaul.